src/main/java/teammates/ui/template/AdminActivityLogTableRow.java

```diff
@@ -145,7 +146,7 @@ public String getLogId() {
     }
 
     public String getActionName() {
-        return activityLog.getActionName();
```

Comment:
Can we rename to `getDisplayedActionName` and move the method above the `Forwarding activityLog methods` as it should belong to `Enhancement to the fields`